botfront/private/defaults/private.dev.yaml

```diff
@@ -19,7 +19,9 @@ policies:
     epochs: 50
     max_history: 5
     batch_size: [32, 64]
-  - name: MemoizationPolicy
+  - name: RulePolicy
+    enable_fallback_prediction: false
+  - name: AugmentedMemoizationPolicy
```

Comment:
@pheel we specifically changed that to MemoizationPolicy to help with tests, isnt that going to cause issue in training tests again ?

Reply:
I had trouble with one test without `AugmentedMemoizationPolicy`. But then again testing now uses `lite-policies.yml` so what we put in here doesn't affect testing.

Reply:
Yeah but lite policies uses the same Augmented. as long as it's the same policy across the board, I think it's fine then